### PR TITLE
Switch sources to use ESM instead of CJS

### PR DESCRIPTION
**Pull request by rvsia**
- [x] https://github.com/RedHatInsights/frontend-components/pull/938
- ~~use esm react table~~

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { useEffect } from 'react';
-import NotificationsPortal from '@redhat-cloud-services/frontend-components-notifications/cjs/NotificationPortal';
-import { Main } from '@redhat-cloud-services/frontend-components/components/cjs/Main';
+import NotificationsPortal from '@redhat-cloud-services/frontend-components-notifications/esm/NotificationPortal';
+import { Main } from '@redhat-cloud-services/frontend-components/components/esm/Main';
```
Comment (Hyperkid123) [inline src/App.js:3]: You could instead upgrade to v3 FEC dependencies which resolve the asset type automatically.

Comment (rvsia) [reply src/App.js:3]: That's exactly what I am planning to do this in follow-up PRs. Now I aiming to switch to ESM, so I can update the wizard package.

File: src/components/SourcesTable/SourcesTable.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import React, { useEffect, useReducer } from 'react';
 import { useDispatch, useSelector, shallowEqual } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-import { Table, TableHeader, TableBody, sortable, wrappable } from '@patternfly/react-table';
+
+import { Table, TableHeader, TableBody, sortable, wrappable } from '@patternfly/react-table/dist/esm/index';
```
Comment (Hyperkid123) [inline src/components/SourcesTable/SourcesTable.js:5]: This is not required. Webpack will use the ESM assets anyway plus the table package does not treeshake react-core anyway :shrug: wont help.

Comment (rvsia) [reply src/components/SourcesTable/SourcesTable.js:5]: Yeah, but I had to use 'cjs' paths in tests for checking table icons. This makes it more transparent.

Comment (rvsia) [reply src/components/SourcesTable/SourcesTable.js]: But yeah, you are right, the mixing is not a big price.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**